mission_control/navigator_launch/launch/simulation.launch

```diff
@@ -38,6 +38,11 @@
         <param name="autorepeat_rate" value="5"/>
     </node>
 
+    <node pkg="navigator_emergency_control" type="navigator_emergency_control" name="navigator_emergency_xbee">
```

Comment:
I think as long as `/wrench/cmd` is outputting the correct values, this should work. All of the buttons use functions in the remote control library, which have been throughly tested. I would recommend placing the launcher for whatever needs to be running on the boat in `gnc.launch` as opposed to hardware_drivers, this is where joystick and keyboard control are brought up. While you're at it, joystick control should be removed from `simulation.launch` as it is no longer used there.